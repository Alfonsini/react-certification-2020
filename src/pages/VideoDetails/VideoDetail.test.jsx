import React from 'react';
import { render, screen } from '../../test/testUtils';

import VideoDetail from '.';

describe('Video detail', () => {
  test('Video detail render correctly', () => {
    render(<VideoDetail />);
    const row = screen.getByTestId('row');
    expect(row).toBeInTheDocument();
  });
});
