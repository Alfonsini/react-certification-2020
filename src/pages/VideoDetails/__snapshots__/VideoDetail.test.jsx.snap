// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video details Video details page render correctly from Home page. Testing hover over clip video. 1`] = `
CSSStyleDeclaration {
  "0": "font",
  "1": "text-rendering",
  "10": "align-items",
  "11": "cursor",
  "12": "color",
  "13": "padding",
  "14": "background-color",
  "15": "box-sizing",
  "16": "opacity",
  "17": "width",
  "18": "height",
  "19": "border",
  "2": "letter-spacing",
  "20": "position",
  "21": "top",
  "22": "right",
  "23": "visibility",
  "3": "word-spacing",
  "4": "line-height",
  "5": "text-transform",
  "6": "text-indent",
  "7": "text-shadow",
  "8": "display",
  "9": "text-align",
  "_importants": Object {
    "align-items": "",
    "background-color": "",
    "border": "",
    "box-sizing": "",
    "color": "",
    "cursor": "",
    "display": "",
    "font": "",
    "height": "",
    "letter-spacing": "",
    "line-height": "",
    "margin": "",
    "opacity": "",
    "padding": "",
    "position": "",
    "right": "",
    "text-align": "",
    "text-indent": "",
    "text-rendering": "",
    "text-shadow": "",
    "text-transform": "",
    "top": "",
    "visibility": undefined,
    "width": "",
    "word-spacing": "",
  },
  "_length": 24,
  "_onChange": [Function],
  "_values": Object {
    "align-items": "flex-start",
    "background-color": "ButtonFace",
    "border": "0px",
    "border-bottom": "0px",
    "border-bottom-color": "buttonface",
    "border-bottom-style": "outset",
    "border-bottom-width": "0px",
    "border-left": "0px",
    "border-left-color": "buttonface",
    "border-left-style": "outset",
    "border-left-width": "0px",
    "border-right": "0px",
    "border-right-color": "buttonface",
    "border-right-style": "outset",
    "border-right-width": "0px",
    "border-top": "0px",
    "border-top-color": "buttonface",
    "border-top-style": "outset",
    "border-top-width": "0px",
    "border-width": "0px",
    "box-sizing": "border-box",
    "color": "ButtonText",
    "cursor": "pointer",
    "display": "inline-block",
    "font": "-webkit-small-control",
    "font-family": "-webkit-small-control",
    "height": "23px",
    "letter-spacing": "normal",
    "line-height": "normal",
    "opacity": "1",
    "padding": "0px",
    "padding-bottom": "0px",
    "padding-left": "0px",
    "padding-right": "0px",
    "padding-top": "0px",
    "position": "absolute",
    "right": "0px",
    "text-align": "center",
    "text-indent": "0",
    "text-rendering": "auto",
    "text-shadow": "none",
    "text-transform": "none",
    "top": "0px",
    "visibility": "visible",
    "width": "23px",
    "word-spacing": "normal",
  },
}
`;

exports[`Video details Video details page render correctly from Home page. Testing hover over clip. 1`] = `
CSSStyleDeclaration {
  "0": "font",
  "1": "text-rendering",
  "10": "align-items",
  "11": "cursor",
  "12": "color",
  "13": "padding",
  "14": "background-color",
  "15": "box-sizing",
  "16": "opacity",
  "17": "width",
  "18": "height",
  "19": "border",
  "2": "letter-spacing",
  "20": "position",
  "21": "top",
  "22": "right",
  "23": "visibility",
  "3": "word-spacing",
  "4": "line-height",
  "5": "text-transform",
  "6": "text-indent",
  "7": "text-shadow",
  "8": "display",
  "9": "text-align",
  "_importants": Object {
    "align-items": "",
    "background-color": "",
    "border": "",
    "box-sizing": "",
    "color": "",
    "cursor": "",
    "display": "",
    "font": "",
    "height": "",
    "letter-spacing": "",
    "line-height": "",
    "margin": "",
    "opacity": "",
    "padding": "",
    "position": "",
    "right": "",
    "text-align": "",
    "text-indent": "",
    "text-rendering": "",
    "text-shadow": "",
    "text-transform": "",
    "top": "",
    "visibility": undefined,
    "width": "",
    "word-spacing": "",
  },
  "_length": 24,
  "_onChange": [Function],
  "_values": Object {
    "align-items": "flex-start",
    "background-color": "ButtonFace",
    "border": "0px",
    "border-bottom": "0px",
    "border-bottom-color": "buttonface",
    "border-bottom-style": "outset",
    "border-bottom-width": "0px",
    "border-left": "0px",
    "border-left-color": "buttonface",
    "border-left-style": "outset",
    "border-left-width": "0px",
    "border-right": "0px",
    "border-right-color": "buttonface",
    "border-right-style": "outset",
    "border-right-width": "0px",
    "border-top": "0px",
    "border-top-color": "buttonface",
    "border-top-style": "outset",
    "border-top-width": "0px",
    "border-width": "0px",
    "box-sizing": "border-box",
    "color": "ButtonText",
    "cursor": "pointer",
    "display": "inline-block",
    "font": "-webkit-small-control",
    "font-family": "-webkit-small-control",
    "height": "23px",
    "letter-spacing": "normal",
    "line-height": "normal",
    "opacity": "1",
    "padding": "0px",
    "padding-bottom": "0px",
    "padding-left": "0px",
    "padding-right": "0px",
    "padding-top": "0px",
    "position": "absolute",
    "right": "0px",
    "text-align": "center",
    "text-indent": "0",
    "text-rendering": "auto",
    "text-shadow": "none",
    "text-transform": "none",
    "top": "0px",
    "visibility": "visible",
    "width": "23px",
    "word-spacing": "normal",
  },
}
`;

exports[`Video details Video details page render correctly from Home page. Testing hover over video clip. 1`] = `
CSSStyleDeclaration {
  "0": "font",
  "1": "text-rendering",
  "10": "align-items",
  "11": "cursor",
  "12": "color",
  "13": "padding",
  "14": "background-color",
  "15": "box-sizing",
  "16": "opacity",
  "17": "width",
  "18": "height",
  "19": "border",
  "2": "letter-spacing",
  "20": "position",
  "21": "top",
  "22": "right",
  "23": "visibility",
  "3": "word-spacing",
  "4": "line-height",
  "5": "text-transform",
  "6": "text-indent",
  "7": "text-shadow",
  "8": "display",
  "9": "text-align",
  "_importants": Object {
    "align-items": "",
    "background-color": "",
    "border": "",
    "box-sizing": "",
    "color": "",
    "cursor": "",
    "display": "",
    "font": "",
    "height": "",
    "letter-spacing": "",
    "line-height": "",
    "margin": "",
    "opacity": "",
    "padding": "",
    "position": "",
    "right": "",
    "text-align": "",
    "text-indent": "",
    "text-rendering": "",
    "text-shadow": "",
    "text-transform": "",
    "top": "",
    "visibility": undefined,
    "width": "",
    "word-spacing": "",
  },
  "_length": 24,
  "_onChange": [Function],
  "_values": Object {
    "align-items": "flex-start",
    "background-color": "ButtonFace",
    "border": "0px",
    "border-bottom": "0px",
    "border-bottom-color": "buttonface",
    "border-bottom-style": "outset",
    "border-bottom-width": "0px",
    "border-left": "0px",
    "border-left-color": "buttonface",
    "border-left-style": "outset",
    "border-left-width": "0px",
    "border-right": "0px",
    "border-right-color": "buttonface",
    "border-right-style": "outset",
    "border-right-width": "0px",
    "border-top": "0px",
    "border-top-color": "buttonface",
    "border-top-style": "outset",
    "border-top-width": "0px",
    "border-width": "0px",
    "box-sizing": "border-box",
    "color": "ButtonText",
    "cursor": "pointer",
    "display": "inline-block",
    "font": "-webkit-small-control",
    "font-family": "-webkit-small-control",
    "height": "23px",
    "letter-spacing": "normal",
    "line-height": "normal",
    "opacity": "1",
    "padding": "0px",
    "padding-bottom": "0px",
    "padding-left": "0px",
    "padding-right": "0px",
    "padding-top": "0px",
    "position": "absolute",
    "right": "0px",
    "text-align": "center",
    "text-indent": "0",
    "text-rendering": "auto",
    "text-shadow": "none",
    "text-transform": "none",
    "top": "0px",
    "visibility": "visible",
    "width": "23px",
    "word-spacing": "normal",
  },
}
`;
